sdk/src/utils/mime.rs

```diff
@@ -18,12 +18,12 @@ pub fn extension_to_mime(extension: &str) -> Option<&'static str> {
         "png" => "image/png",
         "gif" => "image/gif",
         "psd" => "image/vnd.adobe.photoshop",
-        "tiff" => "image/tiff",
+        "tiff" | "tif" => "image/tiff",
```

Comment:
There are formats missing like the avi

Reply:
I'll take a look

Reply:
I added avi , video/avi mappings. Eventually I'd like to generate these mappings from the asset_io drivers somehow.